x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/request/cohere/CohereEmbeddingsRequest.java

```diff
@@ -65,7 +63,12 @@ public HttpRequestBase createRequest() {
         httpPost.setHeader(HttpHeaders.CONTENT_TYPE, XContentType.JSON.mediaType());
         httpPost.setHeader(createAuthBearerHeader(account.apiKey()));
 
-        return httpPost;
+        return new HttpRequest(httpPost, getModelId());
+    }
+
+    @Override
+    public String getModelId() {
+        return modelId;
```

Comment:
is this an inferenceEntityId or a modelVariantId?

Reply:
`inferenceEntityId`